scripts/deploy.sh

```diff
@@ -22,8 +22,14 @@ function update_latest {
     local out_path=$1
     local latest=($(ls -v $out_path | egrep '^('$VERSION_REGEX')$' | tail -1))
     if [ "$latest" == "" ]; then latest='master'; fi
-    rm -f $out_path/latest
-    ln -s $latest $out_path/latest
+
+    if [[ "$latest" =~ ^[^-]+$ ]]; then
+        rm -f $out_path/latest
+        ln -s $latest $out_path/latest
+    fi
+
+    rm -f $out_path/next
+    ln -s $latest $out_path/next
```

Comment:
Does this mean /next will get overwritten on master?

Reply:
No. `latest` is defined as the most recently created directory whose name is a version number, so will never be `master` here

Reply:
Updating `next` should be in else. For situation like: release v3-alpha, then v2.9.4

Reply:
I think `next` will still point at v3-alpha in that case since it's the highest version number, which seems like the right thing. If we move it to the `else`, then when v3 final is released `next` will still point at v3-beta, which I think could be confusing